app/screens/home/search/search.tsx

```diff
@@ -85,14 +84,26 @@ const SearchScreen = ({teamId}: Props) => {
         scrollRef.current?.scrollToOffset({offset, animated: true});
     };
 
+    const handleFilterChange = useCallback((filterValue: FileFilter) => {
+        setLoading(true);
+        setFilter(filterValue);
+        const searchParams: FileSearchParams = {
+            terms: lastSearchedValue + filterFileExtensions(filterValue),
+            is_or_search: true,
+        };
+
+        (async () => {
+            const fileResults = await searchFiles(serverUrl, teamId, searchParams);
+            const fileInfosResult = fileResults?.file_infos && Object.values(fileResults?.file_infos);
+            setFileInfos(fileInfosResult?.length ? fileInfosResult : emptyFileResults);
+        })();
```

Comment:
you can just do `useCallback(async (filterValue...)` instead of using this syntax